### PR TITLE
Remove duplicate code

### PR DESCRIPTION
**Pull request by filipnavara**

### CODE REVIEW

File: src/generator.cs [hunk 1]
```diff
@@ -1163,13 +1163,6 @@ string ParameterGetMarshalType (MarshalInfo mai, bool formatted = false)
 				return "UIntPtr";
 		}
 
-		if (mai.Type.Namespace == "System") {
-			if (mai.Type.Name == "nint")
-				return "IntPtr";
-			if (mai.Type.Name == "nuint")
-				return "UIntPtr";
-		}
-
```
Comment (filipnavara) [inline src/generator.cs:1172]: Identical code block is just above it.

**Review by rolfbjarne — APPROVED** [review]
Looks like a git merge problem. Thanks!

**Comment by rolfbjarne** [comment]
Test failures are unrelated (https://github.com/xamarin/maccore/issues/2443).

**Pull request merged**